src/rgw/rgw_common.h

```diff
@@ -1048,6 +1048,7 @@ struct req_state {
    const char *os_auth_token;
    string swift_user;
    string swift_groups;
+   string swift_account;
```

Comment:
I'm not sure whether the scope should be extended right now. However, the access to the tenant name will be very useful for auth mechanisms when it is impossible to get the tenant basing only on _X-Token-Auth_.